src/Functions/multiIf.cpp

```diff
@@ -373,6 +382,47 @@ class FunctionMultiIf final : public FunctionIfBase
         }
     }
 
+    template<typename T, typename S>
+    static void executeInstructionsColumnarForNullable(std::vector<Instruction> & instructions, size_t rows, const MutableColumnPtr & res, const MutableColumnPtr & null_map)
```

Comment:
It is possible to reuse executeInstructionsColumnar by adding a template param <bool nullable> for it ?

Reply:
if reuse executeInstructionsColumnar, should add some if code, maybe not will for the `for` loop